shopinvader_product_stock/models/shopinvader_backend.py

```diff
@@ -26,8 +26,7 @@ class ShopinvaderBackend(models.Model):
         "Product stock field",
         domain=[
             ('ttype', 'in', ['float', 'integer']),
-            ('model_id.model', '=', 'product.product'),
-            ('name', 'in', ('qty_available', 'virtual_available')),
+            ('model', 'in', ['product.product', 'product.template']),
```

Comment:
@Cedric-Pigeon The goal is to limit the list of fields usable for stock level. IMO we should continue to be strict on these available fields. I don't see any motivation to provide the name of the stock field through a reference to an `ir.model.fields`. IMO we should provide this information thanks a selection field. By using a selection field it becomes easier to extend the list of available field names that can be used to provide the stock. 
Indeed, the only usage of this `ir.model.fields` is to provide the name of the field... https://github.com/akretion/odoo-shopinvader/blob/fccbcae806896838f0379ce211af7b228850c39e/shopinvader_product_stock/models/shopinvader_variant.py#L28

Reply:
@lmignon I do not agree. It is not normal to have to extend a selection field just because you want to expose a new stock. They are plenty of community addons (stock_available) who adds stock field and we should being able to use it.

Reply:
It's also a non sense to be able to select any float or integer field to express a stock level. I prefer to be explicit. 
What do you think about @sebastienbeau

Reply:
For this point I agree but I do not have any better idea as you cannot forecast how people will name their field...

Reply:
OK the most simple solution is the one provided by @Cedric-Pigeon into this PR. We are aware of these drawbacks and advantages but I prefer to keep the code with the most significant informations according to the context know context. IOW the idea to put the field name of potentially used addons is not a good idea.